packages/shared/components/popups/Index.svelte

```diff
@@ -93,6 +94,7 @@
         activityDetails: ActivityDetailsPopup,
         addNode: AddNodePopup,
         aliasConfirmation: AliasConfirmationPopup,
+        authenticationRequired: AuthenticationRequiredPopup,
```

Comment:
It might be better to have this named e.g. `nodeAuthRequired`, something specifically indicating it's for the node since we also have "profile auth".